addons/dockable_container/dockable_container.gd

```diff
@@ -234,7 +242,10 @@ func get_tab_count() -> int:
 
 
 func _can_handle_drag_data(data) -> bool:
-	if data is Dictionary and data.get("type") == "tabc_element":
+	var data_type := "tab_container_tab"
+	if _godot_version.major <= 4 and _godot_version.minor <= 1:  # Earlier than Godot 4.2
+		data_type = "tabc_element"
+	if data is Dictionary and data.get("type") == data_type:
```

Comment:
Instead of checking for Godot version and choosing which key to use, we could check for both, something like the following:
```gdscript
if data is Dictionary and data.get("type") in ["tab_container_tab", "tabc_element"]:
```